Swift/CoreTests/CoreTests-Bridging-Header.h

```diff
@@ -13,6 +13,12 @@
 //  Use this file to import your target's public headers that you would like to expose to Swift.
 //
 #include "ethereum/BREthereum.h"
+#include "crypto/BRCryptoAccount.h"
+#include "crypto/BRCryptoAmount.h"
+#include "crypto/BRCryptoCurrency.h"
+#include "crypto/BRCryptoNetwork.h"
+#include "crypto/BRCryptoPrivate.h"
+#include "crypto/BRCryptoUnit.h"
```

Comment:
This is a preemptive listing of headers?  Only 'Network' and 'Account' are actually needed?

Reply:
The other ones are used to flush out a network (base unit, currency, etc.).